src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_NUMBER = 10;
+    private K[] elementsK;
+    private V[] elementsV;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```

not informative variable names